pkg/controller/podautoscaler/horizontal.go

```diff
@@ -553,6 +553,7 @@ func (a *HorizontalController) computeStatusForObjectMetric(specReplicas, status
 		*status = autoscalingv2.MetricStatus{
 			Type: autoscalingv2.ObjectMetricSourceType,
 			Object: &autoscalingv2.ObjectMetricStatus{
+				DescribedObject: metricSpec.Object.DescribedObject,
```

Comment:
I think a better way to avoid this in the future would be construct the part of the `*status` object that is in common between both value and average value flavors. Are you up for that @mchtech (thx for your patience btw!)

Reply:
@mchtech are you available to do this change?

Reply:
yes, I have constructed the common part of the `*status` object in latest commit. (and rebase to master branch)